CHANGELOG.md

```diff
@@ -1,18 +1,25 @@
 ## [2.23.1] (Unreleased)
 
+BREAKING CHANGES:
+
+ - Raise minimum VS Code version from 1.61.1 to 1.65.2 ([#1176](https://github.com/hashicorp/vscode-terraform/pull/1176))
+ - Improve Extension Settings to follow VS Code setting naming conventions ([#1156](https://github.com/hashicorp/vscode-terraform/pull/1156))
+
 BUG FIXES:
 
  - Ensure extension is installed in remote contexts automatically ([#1163](https://github.com/hashicorp/vscode-terraform/pull/1163))
 
 ENHANCEMENTS:
 
  - Use dark extension icon for preview ([#1143](https://github.com/hashicorp/vscode-terraform/pull/1143))
- - Improve Extension Settings to follow VS Code setting naming conventions ([#1156](https://github.com/hashicorp/vscode-terraform/pull/1156))
  - Set proper scope for machine based settings ([#1164](https://github.com/hashicorp/vscode-terraform/pull/1164))
  - Introduce support for LSP over TCP, with port configurable via `terraform.languageServer.tcp.port` ([#755](https://github.com/hashicorp/vscode-terraform/pull/755))
+ - New Terraform View side bar ([#1171](https://github.com/hashicorp/vscode-terraform/pull/1171))
 
 INTERNAL:
 
+ - Scope languageServer enablement commands [#1178](https://github.com/hashicorp/vscode-terraform/pull/1178))
```

Comment:
```suggestion
 - Only show language server related commands when they're relevant [#1178](https://github.com/hashicorp/vscode-terraform/pull/1178))
```
On a separate note, I think this is worth moving to ENHANCEMENTS as it's a (positive) user-visible change.

Reply:
Agreed!